graphics/opengl/debug.cpp

```diff
@@ -39,10 +39,19 @@ Common::String getGLErrStr(GLenum error) {
 		return "GL_INVALID_VALUE";
 	case GL_INVALID_OPERATION:
 		return "GL_INVALID_OPERATION";
+// OpenGL ES 2.0 doesn't have GL_STACK_OVERFLOW and GL_STACK_UNDERFLOW
```

Comment:
Could you define these in `graphics/opengl/system_headers.h` instead?

Reply:
The reason I excluded these here is that they aren't returned in GLES 2.0 (see https://registry.khronos.org/OpenGL-Refpages/es2.0/xhtml/glGetError.xml)

But happy to adjust if you think it's clearer to add in.  (or maybe the comment is not precise?)

Reply:
You can't use _KHR defines here as it would break the build for platforms not having this define (because it's an extension) and not using GLAD.
I think that what @ccawley2011 proposes is to replicate these defines in `graphics/opengl/system_headers.h` around line 85 to make sure they exist.

Reply:
Thank you for the clarification, I added them now to the system headers.